### DISCUSSION TITLE
Disable prefetch on Link component when hovered

### DISCUSSION THREAD

**Discussion by hk86** [vercel/next.js#24120] [Help]
Hi there,

im using `import Link from 'next/link'` and next-pwa. The problem is that whenever a Link component is hovered the target page json and json is loaded:

![image](https://user-images.githubusercontent.com/32885031/114941816-e9f3d280-9e43-11eb-9b8e-314b81371a14.png)

How can I disable this behavior? I have a site with lots of visitors and it is consuming a ton of bandwidth.

Best regards,

HK

**Comment by hk86** [comment]
Any ideas?

**Comment by stigkj** [comment]
Have you tried the `prefetch` option on the `Link` component?

**Comment by hk86** [comment]
I've set it to `False`. It's not prefetching the page on sight, but still it is prefetched when the user hovers the Link.

**Reply by nihakue** [reply]
It doesn't look like there's any way to disable this behavior: https://github.com/vercel/next.js/blob/canary/packages/next/client/link.tsx#L284

which seems odd.

**Reply by hk86** [reply]
Thanks! I made a local copy of the link component and disabled this functionality.

It would sense to add a new prop in order to be able to disable the prefetching.
